docs/sources/mimir/release-notes/v2.14.md

```diff
@@ -0,0 +1,139 @@
+
+Incoming OTLP requests were previously size-limited with the distributor's `-distributor.max-recv-msg-size` configuration.
+Distributor has a new `-distributor.max-otlp-request-size` configuration for limiting OTLP requests. The default value is set to limit request to 100 MiB.
+
+Distributor now replies with `Retry-After` header on retryable errors. The feature is controlled by the `-distributor.retry-after-header.enabled` configuration,
+that is enabled by default.
+
+## Important changes
+
+In Grafana Mimir 2.14 the following behavior has changed:
+
+When executing a remote read request, the querier now honors the start/end time range, specified in the read hints.
+
+To prevent queue starvation with querier-worker queue prioritization algorithm, the querier now sets the minimum `-querier.max-concurrent` to four.
+Values below the allowed minimum are ignored.
+
+Distributor configuration's `-distributor.retry-after-header.max-backoff-exponent` and `-distributor.retry-after-header.base-seconds` are replaced
+with `-distributor.retry-after-header.min-backoff` and `-distributor.retry-after-header.max-backoff` for easier configuration.
+
+The circuit breakers in ingester are improved to not open in case of per-instance limit errors. Since now on the circuit breakers
+trigger the opening only if push and pull requests exceed the configured duration.
+
+The default inactivity timeout of active series in ingesters, controlled by the `-ingester.active-series-metrics-idle-timeout` configuration
+is increased from `10m` to `20m`.
+
+Query-frontend now returns the "413 Request Entity Too Large" error when a response shard from the `/active_series` request exceeds the limit.
+
+Following featues of store-gateway are changed: `-blocks-storage.bucket-store.max-concurrent-queue-timeout` is set to 5 seconds;
+`-blocks-storage.bucket-store.index-header.lazy-loading-concurrency-queue-timeout` is set to 5 seconds;
+`-blocks-storage.bucket-store.max-concurrent` is set to 200;
+
+Number of previously deprecated configuration options were removed in the release:
+
+- the `-ingester.return-only-grpc-errors` option in ingester
+- the `-ingester.client.circuit-breaker.*` options in ingester
+- the `-ingester.limit-inflight-requests-using-grpc-method-limiter` option in ingester
+- the `-ingester.client.report-grpc-codes-in-instrumentation-label-enabled` option in distributor and ruler
+- the `-distributor.limit-inflight-requests-using-grpc-method-limiter` option in distributor
+- the `-distributor.enable-otlp-metadata-storage` option in distributor
+- the `-ruler.drain-notification-queue-on-shutdown` option in ruler
+- the `-querier.max-query-into-future` option in querier
+- the `-querier.prefer-streaming-chunks-from-store-gateways` option in querier and store-gateway
+- the `-query-scheduler.use-multi-algorithm-query-queue` option in querier-scheduler
```

Comment:
Do we make note of _all_ configuration options removed, or just the stable/non-experimental ones? This one was experimental.